plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/results/ResultsCCUIListener.java

```diff
@@ -1,469 +1,474 @@
-				units.add(new ResultsUnit(className, null, err, false, first));
-				first = false;
-			}
-
-			for (IAnalysisSeed seed : findings.getSeeds()) {
-				if (units.parallelStream().noneMatch(e -> e.doesSeedmatchWithError(seed))) {
-					units.add(new ResultsUnit(className, seed, null, true));
-				}
-			}
-		}
-
-		StatisticsView.allowAnalysisRerun(true);
-		StatisticsView.updateView(stat.getProject(), stat.getTimeOfAnalysis(), units);
-	}
-
-	@Override
-	public void beforeConstraintCheck(final AnalysisSeedWithSpecification analysisSeedWithSpecification) {
-
-	}
-
-	@Override
-	public void afterConstraintCheck(final AnalysisSeedWithSpecification analysisSeedWithSpecification) {
-
-	}
-
-	@Override
-	public void beforePredicateCheck(final AnalysisSeedWithSpecification analysisSeedWithSpecification) {
-
-	}
-
-	@Override
-	public void afterPredicateCheck(final AnalysisSeedWithSpecification analysisSeedWithSpecification) {
-
-	}
-
-	@Override
-	public void seedStarted(final IAnalysisSeed analysisSeedWithSpecification) {
-
-	}
-
-	@Override
-	public void boomerangQueryStarted(final Query seed, final BackwardQuery q) {
-
-	}
-
-	@Override
-	public void boomerangQueryFinished(final Query seed, final BackwardQuery q) {
-
-	}
-
-	@Override
-	public void ensuredPredicates(final Table<Statement, Val, Set<EnsuredCryptSLPredicate>> existingPredicates,
-			final Table<Statement, IAnalysisSeed, Set<CryptSLPredicate>> expectedPredicates, final Table<Statement, IAnalysisSeed, Set<CryptSLPredicate>> missingPredicates) {}
-
-	@Override
-	public void addProgress(final int processSeeds, final int workListsize) {
-		setProcessedSeeds(processSeeds);
-		setTotalSeeds(workListsize + processSeeds);
-		setPercentCompleted((int) Math.round((float) processedSeeds * 100 / totalSeeds));
-		tempWork = getPercentCompleted() - work;
-		if (tempWork > 0) {
-			setWorkUnitsCompleted(tempWork);
-			work = getPercentCompleted();
-		} else {
-			setWorkUnitsCompleted(0);
-		}
-	}
-
-	public int getPercentCompleted() {
-		return percentCompleted;
-	}
-
-	public void setPercentCompleted(int percentCompleted) {
-		this.percentCompleted = percentCompleted;
-	}
-
-	public int getTotalSeeds() {
-		return totalSeeds;
-	}
-
-	public void setTotalSeeds(int totalSeeds) {
-		this.totalSeeds = totalSeeds;
-	}
-
-	public int getProcessedSeeds() {
-		return processedSeeds;
-	}
-
-	public void setProcessedSeeds(int processedSeeds) {
-		this.processedSeeds = processedSeeds;
-	}
-
-	public boolean isCgGenComplete() {
-		return cgGenComplete;
-	}
-
-	public void setCgGenComplete(boolean cgGenComplete) {
-		this.cgGenComplete = cgGenComplete;
-	}
-
-	public int getWorkUnitsCompleted() {
-		return workUnitsCompleted;
-	}
-
-	public void setWorkUnitsCompleted(int workUnitsCompleted) {
-		this.workUnitsCompleted = workUnitsCompleted;
-	}
-
-	public void setWork(int work) {
-		this.work = work;
-	}
-
-}
+/********************************************************************************
```

Comment:
I've not reviewed this file because of how Github shows the diff. Could you please briefly elaborate what you have modified here? Thanks.

Reply:
In this class, I only add this line `errorInfoMap.put("predicateParamCount", ((RequiredPredicateError) error).getContradictedPredicate().getParameters().size()+"");` to check later how many parameter the predicates needs.